blog_system/settings.py

```diff
@@ -0,0 +1,132 @@
+"""
+Django settings for blog_system project.
+
+Generated by "django-admin startproject" using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / "subdir".
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-@)h#)@f2w#0(-gnrc%qx7arb_cro5^-)84+ho26s@vpy8vt@&k"
```

Comment:
The `SECRET_KEY` is exposed in the settings file. It's important to keep this key secret in production environments. Consider using environment variables or a separate configuration file to manage sensitive information.